test/lib/y2firewall/importer_strategies/suse_firewall.rb

```diff
@@ -85,6 +81,24 @@
         expect(firewalld.default_zone).to eql("dmz")
       end
 
+      context "and protection from INT zone is disabled" do
+        it "configures the INT zone as the trusted" do
+          trusted = firewalld.find_zone("trusted")
+
+          expect(trusted.interfaces).to eq(["eth1"])
+        end
+      end
+
+      context "and protection from INT zone is enabled" do
```

Comment:
maybe add also case when it is not defined at all?

Reply:
ok, in that case it is like a disabled or not defined, but can add a separate case

Reply:
I understand it from code, but it should be documented and tested in tests. so separate context make sense for me.